runtime/runtime.m

```diff
@@ -2548,12 +2548,17 @@ -(void) xamarinSetFlags: (enum XamarinGCHandleFlags) flags;
 {
 	char *pinvokeOverride = xamarin_strdup_printf ("%p", &xamarin_pinvoke_override);
 	char *icu_dat_file_path = NULL;
+	int subtractPropertyCount = 0;
 
-	char path [1024];
-	if (!xamarin_locate_app_resource (xamarin_icu_dat_file_name, path, sizeof (path))) {
-		LOG (PRODUCT ": Could not locate the ICU data file '%s' in the app bundle.\n", xamarin_icu_dat_file_name);
+	if (xamarin_icu_dat_file_name != NULL && *xamarin_icu_dat_file_name != 0) {
+		char path [1024];
```

Comment:
Just for me to learn, why 1024?

Reply:
Because we're probably not getting any paths bigger than that.